stock_aggregated_move_line/tests/TEST_INSTRUCTIONS.rst

```diff
@@ -0,0 +1,4 @@
+- Open Inventory > Operations > Transfers and create a new Transfer to Deco Addict
+- Enter two product lines with the same product - Pedal Bin, change the description on one of them
+- Confirm the delivery order and press Print > Delivery Slip
+- The Delivery Slip must include both products with their respective descriptions
```

Comment:
The test seems wrong. The aggregated section should combine the two products to one line.

Reply:
@janikvonrotz I changed the test instructions, in case the two same products are on different product lines, they get aggregated on the purchase order and the delivery slip